src/puppetlabs/puppetdb/scf/storage_utils.clj

```diff
@@ -51,10 +53,19 @@
         minor (.getDatabaseMinorVersion db-metadata)]
     [major minor]))
 
+(defn get-db-metadata
+  []
+  (let [db-metadata (.. (sql/find-connection)
```

Comment:
I think you want to us the undeprecated `sql/` namespace here, passing in `db-config` and calling `get-connection` on the `db-config`.